arbitrary-send-erc20.sol

```diff
@@ -0,0 +1,9 @@
+    // ruleid: arbitrary-send-erc20
+    function a(address from, address to, uint256 amount) public {
+        erc20.transferFrom(address, to, am);

```

Comment:
one more thing: move `//ruleid` to right above line 3 (as that's what our internal testing expects)

Reply:
What is the best way to recommit this file? I used the recommended playground to submit the PR, but am having difficulty making changes to this PR. Thank you!

Reply:
I see! Creating a new PR is probably the best option here, as branch protections make it hard to push changes to existing PRs. Sorry about the inconvenience 😓

Reply:
No problem, can you please delete this PR. It is giving me a conflict error when I try to create a new PR. Thank you for your help!